components/analytics/components/Filter/index.js

```diff
@@ -244,7 +270,7 @@ class Filter extends React.Component {
 								))}
 							</Select>
 						)}
-						{filterKey && (
+						{filterKey && filterKey !== 'custom_event' && (
```

Comment:
@jyash97 What is this?
Can you explain that?

Reply:
Check this line in current code: https://github.com/appbaseio/batteries/blob/arc/components/analytics/components/Filter/index.js#L135

We dont render this Select component on custom_event filter. We just want to render the label filter.